packages/react-components/global.d.ts

```diff
@@ -0,0 +1,5 @@
+import 'jest-extended';
+
+declare global {
+  const IS_REACT_ACT_ENVIRONMENT: boolean;
```

Comment:
Nit: From https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html & https://github.com/testing-library/react-testing-library/issues/509#issuecomment-917989936, we might not need this flag anymore if we have `@testing-library/react` after `13.0.0-alpha.1`. And we are using `^14.0.0` in all packages.

Reply:
will remove in next PR